.github/workflows/publish.yml

```diff
@@ -10,16 +10,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
-    - name: Setup MiniConda
+    - name: Install Conda Deps
```

Comment:
changes reviewed in [PR to this branch](https://github.com/simularium/simulariumio/pull/95)